httpobs/database/schema.sql

```diff
@@ -84,6 +84,7 @@ CREATE MATERIALIZED VIEW latest_scans
   LATERAL ( SELECT id AS scan_id, site_id, state, start_time, end_time, tests_failed, tests_passed, grade, score, error
             FROM scans WHERE site_id = s.id AND state = 'FINISHED' ORDER BY end_time DESC LIMIT 1 ) latest_scans;
 COMMENT ON MATERIALIZED VIEW latest_scans IS 'Most recently completed scan for a given website';
+GRANT SELECT ON latest_scans TO httpobsapi;
```

Comment:
Just trying to be consistent with granting read access to these materialized views, as they only contain useful statistics.